automated_testing/test/pageobjects/index-page.ts

```diff
@@ -123,6 +119,22 @@ class IndexPage extends Page {
         return $(selectors.pageSeePastContestsButtonSelector);
     }
 
+    public get secondCompeteCardButtonActiveContests() {
+        return $(selectors.pageAllActiveCardsSelector).$$(selectors.pageCompeteButtonIdSelector)[1];
+    }
+
+    public get secondPracticeCardButtoPastContests() {
+        return $(selectors.pageAllPastCardsSelector).$$(selectors.pagePracticeButtonIdSelector)[1];
```

Comment:
Isn't this the same logic as in `practiceButtoInPastContestsCard()`?
Maybe just do `practiceButtoInPastContestsCard()[0]``?

Reply:
It is the second practice button. That is why I used `$$(selectors.pagePracticeButtonIdSelector)[1]; `
The same logic goes for the second compete button. I can delete those selectors as they were used when I didn't have a local DB set up.